src/modules/components/dao/daoDataListItem/index.ts

```diff
@@ -0,0 +1 @@
+export { DaoDataListItem, type IDaoDataListItemProps } from './daoDataListItem';
```

Comment:
I would suggest to start structuring these DataListItem component exports in a way that we can later add a `DaoDataListItem.Loader` component to handle the loading elements, something like:
```suggestion
import { DaoDataListItemStructure } from './daoDataListItemStructure';

export const DaoDataListItem = {
    Structure: DaoDataListItemStructure
}

export type { IDaoDataListItemProps } from './daoDataListItem';
```

This means that the folder structure will be like:
```
/daoDataListItem
    index.ts
    /daoDataListItemStructure
        daoDataListItemStructure.tsx
        daoDataListItemStructure.testt.sx
        daoDataListItemStructure.stories.tsx
        index.ts
    /daoDataListItemLoader [to be implemented later]
```

Reply:
@sepehr2github The `/daoDataListItemStructure` folder should be inside a `/daoDataListItem` folder